src/network/networkd-network.h

```diff
@@ -148,6 +148,8 @@ struct Network {
         bool dhcp_use_routes;
         int dhcp_use_gateway;
         bool dhcp_quickack;
+        uint32_t dhcp_initcwnd;
+        uint32_t dhcp_initrwnd;
```

Comment:
It is unexpected that the two names are abbreviated differently, "Advertised" is skipped in the variable name. The variable is used just a few times, and it's better to not abbreviate at all so it's obvious to the reader what this is. Please rename them to `dhcp_initial_congestion_window` and `dhcp_advertised_receive_window`.